data/json/terrain.json

```diff
@@ -7020,5 +7020,54 @@
       "ter_set": "t_pit_shallow",
       "items": [ { "item": "rock", "count": 8 }, { "item": "stick", "count": [ 8, 12 ] } ]
     }
+  },
+  {
+    "type": "terrain",
+    "id": "t_shrub_cranberry",
+    "looks-like": "t_shrub_blueberry",
+    "name": "cranberry shrub",
+    "symbol": "#",
+    "color": "red_green",
```

Comment:
This should be `"light_red_green"`, the unharvested bushes are lighter than the harvested ones.